include/clang/Interpreter/CppInterOp.h

```diff
@@ -444,6 +444,9 @@ namespace Cpp {
   /// Returns the resource-dir path (for headers).
   const char* GetResourceDir();
 
+  /// Get Include Paths
+  void GetIncludePath(const char* dir, std::vector<std::string>& includePaths);
```

Comment:
```suggestion
  void GetIncludePath(std::vector<std::string>& includePaths);
```

This function should only have a single output parameter.

Reply:
Oh okay, I will make the required changes.